src/releasers/node.ts

```diff
@@ -39,9 +38,7 @@ export class Node extends ReleasePR {
     if (pkg.name) this.packageName = pkg.name;
 
     const latestTag: GitHubTag | undefined = await this.gh.latestTag(
-      this.monorepoTags
-        ? `${packageBranchPrefix(this.packageName, 'node')}-`
-        : undefined
+      this.monorepoTags ? `${this.packagePrefix}-` : undefined
```

Comment:
Wasn't immediately following why we can drop the helper `packageBranchPrefix` here.

Also, if the goal is hopefully to move most the releasers over to a stable branch name, I wonder if the idea of a `prefix` makes sense, should we just be passing in the full branch name?

Reply:
It's been appropriately moved over into a new `ReleasePR.coercePackagePrefix` - I like that change! However, see my note on the tests as I think it's not quite all the way there yet for Node.

also, in the spirit of "Small CLs" that change could probably have been pulled out as a separate PR ahead of this change.